test/dutchExchange-DepositAndSell.js

```diff
@@ -0,0 +1,68 @@
+/* eslint no-floating-decimal:0 */
+const {
+  timestamp,
+} = require('./utils')
+
+const {
+  getContracts,
+  getAuctionIndex,
+} = require('./testFunctions')
+
+// Test VARS
+let eth
+let gno
+let dx
+
+
+let contracts
+
+contract('DutchExchange deposit/withdraw tests', (accounts) => {
+  const [master] = accounts
+  const testingAccs = accounts.slice(1, 5)
+
+  const ETHBalance = 10..toWei()
+
+  // const GNOBalance = 40..toWei()
+
+  before(async () => {
+    // get contracts
+    contracts = await getContracts();
+    // destructure contracts into upper state
+    ({
+      DutchExchange: dx,
+      EtherToken: eth,
+      TokenGNO: gno,
+    } = contracts)
+
+    await Promise.all(testingAccs.map(acc => Promise.all([
+      eth.deposit({ from: acc, value: ETHBalance }),
+      eth.approve(dx.address, ETHBalance, { from: acc }),
+    ])))
+
+    await Promise.all(testingAccs.map(acc => Promise.all([
+      dx.deposit(eth.address, ETHBalance / 2, { from: acc }),
+    ])))
```

Comment:
Can do without the second `Promise.all` here.

Reply:
yep, refactored this and didn't catch it. thx